atlassian_jwt_auth/contrib/aiohttp/key.py

```diff
@@ -21,8 +22,16 @@ def _get_session(self):
         return aiohttp.ClientSession(loop=self.loop)
 
     async def _retrieve(self, url, requests_kwargs):
```

Comment:
Unrelated, but is this library meant to support Python 2?

Edit: never mind, I now see how this is used only on p3k.